packages/utilities/fast-react-wrapper/src/index.ts

```diff
@@ -242,109 +243,114 @@ export function provideReactWrapper(React: any, designSystem?: DesignSystem) {
             type = type.type as any;
         }
 
-        class ReactComponent extends React.Component<InternalProps> {
-            private _element: TElement | null = null;
-            private _elementProps!: { [index: string]: unknown };
-            private _userRef?: ReactModule.Ref<unknown>;
-            private _ref?: (element: TElement | null) => void;
+        if (!wrappersCache.has(type)) {
```

Comment:
I'm not sure if it's that much more efficient or not, but you could also just do a `get` here and if it is not undefined, just return the result, otherwise create the wrapper and `set` it. This would eliminate needing to have the `has` call.

Reply:
Love it, way cleaner I think!